doc/providers/totp.rst

```diff
@@ -93,7 +93,7 @@ and the period of the temporary codes.
            public function getTotpAuthenticationConfiguration(): ?TotpConfigurationInterface
            {
                // You could persist the other configuration options in the user entity to make it individual per user.
-               return new TotpConfiguration($this->totpSecret, TotpConfiguration::ALGORITHM_SHA1, 20, 8);
+               return $this->isTotpAuthenticationEnabled() ? new TotpConfiguration($this->totpSecret, TotpConfiguration::ALGORITHM_SHA1, 20, 8) : null;
```

Comment:
I see the point. Though if the secret being null is the issue, better check for `$this->totpSecret` instead of $this->isTotpAuthenticationEnabled(). That makes is more explicit what is going on.

```suggestion
               return null !== $this->totpSecret ? new TotpConfiguration($this->totpSecret, TotpConfiguration::ALGORITHM_SHA1, 20, 8) : null;
```